src/services/jwtService.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const jwt = require('jsonwebtoken');
+
+const generateAccessToken = user =>
+  jwt.sign(user, process.env.JWT_SECRET, { expiresIn: '10m' });
```

Comment:
You are using the same JWT_SECRET to sign both access and refresh tokens. For better security, consider using different secrets for access and refresh tokens.